src/commands/mobile_app/upload.rs

```diff
@@ -0,0 +1,26 @@
+use anyhow::Result;
+use clap::ArgAction;
+use clap::{Arg, ArgMatches, Command};
+
+use crate::utils::args::ArgExt;
+
+pub fn make_command(command: Command) -> Command {
+    command
+        .about("Upload mobile app files to a project.")
+        .org_arg()
+        .project_arg(false)
+        .arg(
+            Arg::new("paths")
+                .value_name("PATH")
+                .help("The path to the mobile app files to upload. Supported files include Apk, Aab or XCArchive.")
+                .num_args(1..)
+                .action(ArgAction::Append),
+        )
+}
+
+#[allow(clippy::unnecessary_wraps)]
+pub fn execute(_matches: &ArgMatches) -> Result<()> {
+    println!("Uploading mobile app files to a project is not yet implemented.");
+
```

Comment:
[nit] I would use `eprintln!` here so that this message goes to `stderr` rather than `stdout`.

```suggestion
    eprintln!("Uploading mobile app files to a project is not yet implemented.");
```